src/chatClient.cpp

```diff
@@ -1852,6 +1855,7 @@ promise::Promise<void> GroupChatRoom::decryptTitle()
         wptr.throwIfDeleted();
         KR_LOG_ERROR("Error decrypting chat title for chat %s:\n%s\nFalling back to member names.", karere::Id(chatid()).toString().c_str(), err.what());
         makeTitleFromMemberNames();
+        return err;
```

Comment:
@alxvasilev It is enough with commit abea0588d725dea1fd1d28eb9d6bb5b910d5cbc6 or is something else necesary?
I modify the code to adapt to convention